packages/edit-site/src/components/page-patterns/fields.js

```diff
@@ -133,8 +133,7 @@ function TitleField( { item } ) {
 					title
 				) : (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> check for patterns and the pattern title.

**Note:** It has a link variant, so there is an override style present, so before and after is same.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="page-patterns-before" src="https://github.com/user-attachments/assets/552cbd1f-4619-40e2-a7c5-d904197a8173"> | <img width="419" alt="page-patterns-after" src="https://github.com/user-attachments/assets/656c0f06-2a82-46df-a6db-273930e01395"> |